src/views/Vote/VoteForm/VoteForm.vue

```diff
@@ -75,13 +79,15 @@ export default class VoteForm extends Vue {
   }
 
   private async handleSubmit() {
+    if (this.voteScore) {
```

Comment:
You could also have used:

```typescript
const isCreated = await this.createVote({
...this.input,
votingId: this.voting.id,
groupId: this.group.id,
score: this.voteScore,
})
```

If `voteScore` doesn't have a value, does this.input.score have a value ?